File: src/main/java/org/mifos/fineract/services/MixMapping_Api.java
package org.mifos.fineract.services;

import retrofit2.http.GET;
import retrofit2.http.Headers;
import retrofit2.http.PUT;
import rx.Observable;

public interface MixMapping_Api {
    /**
     * @return Observable&lt;String&gt;
     */
    @Headers({
            "Content-Type:application/json"
    })
    @GET("mixmapping")
    Observable<String> retrieveTaxonomyMapping();


    /**
     * @param body (optional)
     * @return Observable&lt;String&gt;
     */
    @Headers({
            "Content-Type:application/json"
    })
    @PUT("mixmapping")
    Observable<String> updateTaxonomyMapping(
            @retrofit2.http.Body String body
    );

}

File: src/main/kotlin/org/openapitools/client/apis/MixMappingApi.kt
package org.openapitools.client.apis

import retrofit2.http.Body
import retrofit2.http.GET
import retrofit2.http.PUT


interface MixMappingApi {
    /**
     *
     *
     * Responses:
     *  - 0: default response
     *
     * @return [kotlin.String]
     */
    @GET("v1/mixmapping")
    suspend fun retrieveTaxonomyMapping(): String

    /**
     *
     *
     * Responses:
     *  - 0: default response
     *
     * @param body  (optional)
     * @return [kotlin.String]
     */
    @PUT("v1/mixmapping")
    suspend fun updateTaxonomyMapping(@Body body: String? = null): String

}
